firechat/accounts/models.py

```diff
@@ -20,14 +25,13 @@ def get(self, *args, **kwargs):
         return super().prefetch_related("following", "followers").get(*args, **kwargs)
 
 
-class User(AbstractUser):
+class User(AbstractUser, ActivatorModel):
     profile = models.ImageField(upload_to=_upload_to, null=True, blank=True)
     age = models.IntegerField(null=True, blank=True)
     address = models.CharField(max_length=128, null=True, blank=True)
     phone = PhoneNumberField(region="IN", blank=True, null=True)
     streak = models.IntegerField(default=0)
-    status = models.IntegerField(choices=STATUS_CHOICES, default=INACTIVE_STATUS)
-    last_streak_update = models.DateTimeField(blank=True, null=True, auto_now_add=True)
+    last_streak_update = CreationDateTimeField()
```

Comment:
You can also use timestamp model so from which you will get updated_at

Reply:
I did not use the Timestamp model on purpose, Since I Used the CreationDateTimeField() which I imported from django_extensions and did not require 2 fields of the Timestamp Model. Also now I can use relevant field name.